explorer/src/utils/anchor.tsx

```diff
@@ -52,14 +28,9 @@ export function ProgramName({
 }) {
   const defaultProgramName =
     programLabel(programId.toBase58(), cluster) || "Unknown Program";
-
-  return (
-    <React.Suspense fallback={defaultProgramName}>
-      <ErrorBoundary fallback={<>{defaultProgramName}</>}>
-        <AnchorProgramName programId={programId} url={url} />
-      </ErrorBoundary>
-    </React.Suspense>
```

Comment:
Minus the `ErrorBoundary` I think this was the right abstraction. Devs can just render a `<ProgramName>` without having to worry about Suspense or supplying a default while the presence of an Anchor program is determined.

This change has created more work for callers (eg. `explorer/src/components/ProgramLogsCardBody.tsx` above) where for a time you'll have the string ` Instruction` with a leading space until you have an answer.

Does that make sense?

Reply:
100%. I reverted to something closer to the original. Lmk if that works.